kolibri/core/tasks/api.py

```diff
@@ -314,11 +393,15 @@ def queues(self):
         return [queue, priority_queue]
 
     def default_permission_classes(self):
-        # exclusive permission for facility management
-        if self.action == "startexportlogcsv":
+        if self.action in ["list", "deletefinishedtasks"]:
```

Comment:
Presumably this would all just go away when we transfer all these permissions over to the tasks themselves.

Reply:
Yep that's true but for the time being this change fixes an issue we discussed on Slack.